chat impr

djordjenikolic960 opened the pull request:
- if user has image then it is set, else first letter of its name

djordjenikolic960 committed:
- if user has image then it is set, else first letter of its name

Teahaf commented on app/src/main/java/com/example/demoappforfirebase/Fragment/ChatFragment.kt:
```diff
@@ -60,6 +62,29 @@ class ChatFragment : BaseFragment() {
+            }
+
+            override fun onCancelled(error: DatabaseError) {
+                TODO("Not yet implemented")
```
Analytics

Teahaf commented on app/src/main/java/com/example/demoappforfirebase/Fragment/ChatFragment.kt:
```diff
@@ -60,6 +62,29 @@ class ChatFragment : BaseFragment() {
+            }
+        })
+
+        val userQuery = database.child("Users").child(preferencesHelper.getUserId())
```
mozda bi mogao da se napravi current user objekat u nekom od VM-ova, jer se koristi na puno mesta, pa da se to refaktorise
Ako ti se ne radi sada, samo ga zapisi kao task

Teahaf commented on app/src/main/java/com/example/demoappforfirebase/Fragment/ChatFragment.kt:
```diff
@@ -60,6 +62,29 @@ class ChatFragment : BaseFragment() {
+            }
+
+            override fun onCancelled(error: DatabaseError) {
+                TODO("Not yet implemented")
```
isto analytics

Teahaf commented on app/src/main/java/com/example/demoappforfirebase/Fragment/ChatFragment.kt:
```diff
@@ -135,24 +159,27 @@ class ChatFragment : BaseFragment() {
         } else {
-            //todo pribaviti korisnika, proveriti da li ima sliku ako ima setovati nju, ako nema prvo slovo njegovog imena
+            if (chatVM.otherUser?.picture == "") {
+                view.userNameFirstLetter.text = chatVM.otherUser?.name?.first()?.toUpperCase().toString()
```
mozda bi mogli da koristimo ovo i za profilnu

djordjenikolic960 committed:
Merge remote-tracking branch 'origin/master' into userImageInChat

djordjenikolic960 committed:
- chat bug fixed
- layout for sending message moved to content main
- code refactored
- user that is log in moved to userVM

djordjenikolic960 committed:
- forgot to check main activity when sending

Teahaf reviewed (approved):
sve kul
